docs/src/_cloud/onboarding.md

```diff
@@ -52,44 +38,87 @@ Once you've completed the above, you are ready to onboard your project to Meltan
-1. Preferred Organization ID ("org ID") on Meltano Cloud
-   - Organization ID :
-     - must be globally unique
-     - must be 20 characters or less
-     - can contain a combination of lowercase letters, numbers, and dashes (no other special characters).
-   - E.g. `abc-company`, `abc-company-usa`, `abc-data-team`, `abc-finance-team`, etc.
-1. For each project:
-   1. Git repo information:
-      1. Name of hosting provider (e.g. GitHub, GitLab, etc.)
-      1. Git repo URL
-      1. Git branch name
-      1. The name of the [Meltano Environment](/concepts/environments) to use for onboard and testing purposes.
-         - _Recommended environment name is `'sandbox'`. See prereqs above for more information._
-   1. For each schedule you would like to run in Meltano Cloud:
-      1. Schedule name
-1. Per authorized user:
-   1. User's Full Name
-   1. User's Email Address
-   1. User's GitHub ID (used for identity, authentication, and permissioning)
-   1. User's Role: `owner`, `maintaner`, or `reader`
-      - See [users and roles](/platform/#roles-and-permissions) for more information.
-
-### Step 2: Encrypt and submit the `.env` file
-
-After receiving the above information, Meltano will register your project(s) and generate a new set of encryption and decryption keys specific to your organization. We will then provide you with your organization's public key along with instructions to encrypt your `.env` file and attach your encrypted file to your project.
-
-The [kms-ext tool](https://github.com/meltano/kms-ext) is available to use for the encryption process.
-See the [encrypting secrets docs](/encrypting_secrets) for more details and examples of how to do this.
-Also be aware that there are some [reserved variables](/platform/#reserved-variables) only for use by Meltano Cloud.
-
-Also see the [security whitepaper](/security) for more information on encryption algorithms.
-
-### Step 3: Initial execution and debugging
-
-Once the above steps are complete, Meltano will initialize your account and execute the provided schedules. If any errors occur during execution, we will notify you and provide access to the necessary logs for debugging.
+1. Primary contact name and email: `___________`
+1. GitHub repo information:
+   1. Repo URL: `___________`
+   2. Initial Meltano Environment Name: `___________` (e.g. 'prod' or 'staging')
+   3. Git branch name to use for the above environment: `___________` (e.g. `main`)
+1. List of GitHub User IDs, separated by commas: `___________`
+1. List of Meltano schedule names and intervals: `___________`
+   - This can be copy-pasted from `meltano.yml`.
+
+### Step 2: Install (or update) the Cloud CLI
+
+To install from scratch using [pipx](https://pypa.github.io/pipx/installation/#install-pipx):
+
+```console
+pipx install 'git+https://github.com/meltano/meltano.git@cloud#subdirectory=src/cloud-cli'
+```
+
+To upgrade to the latest version:
+
+```console
+pipx reinstall meltano-cloud-cli
```

Comment:
Does `pipx upgrade meltano-cloud-cli` not work with git-based installations?

Reply:
More precisely, it becomes a no-op if there isn't a version bump.